packages/kbn-test/src/kbn_client/kbn_client_requester_error.ts

```diff
@@ -18,6 +18,7 @@ export class KbnClientRequesterError extends Error {
 }
 function clean(error: Error): AxiosError {
   const _ = AxiosError.from(error);
+  delete _.cause;
```

Comment:
The `AxiosError.from` function sets cause to the entire incoming `error` object. Opted to delete `cause` to address this concern: https://github.com/elastic/kibana/pull/202774#discussion_r1869061750